internal/tiltfile/local_resource.go

```diff
@@ -66,9 +68,17 @@ func (s *tiltfileState) localResource(thread *starlark.Thread, fn *starlark.Buil
 		return nil, err
 	}
 
+	updateCmd := model.ToShellCmd(updateCmdStr)
+	serveCmd := model.ToShellCmd(serveCmdStr)
+
+	if updateCmd.Empty() && serveCmd.Empty() {
+		return nil, fmt.Errorf("local_resource must have an updateCmd or a serveCmd, but both were empty")
+	}
+
 	res := localResource{
 		name:         name,
-		cmd:          model.ToShellCmd(cmd),
+		updateCmd:    model.ToShellCmd(updateCmdStr),
```

Comment:
can just use `updateCmd` and `serveCmd` here

Reply:
whoops, yep that was a refactoring mistake